test/triangle-test.js

```diff
@@ -0,0 +1,30 @@
+var assert = require('chai').assert;
+var isTriangle = require('../triangle')
+
+describe("it knows if it is a real triangle", function() {
+  it("returns true when sides make a rational triangle", function(){
+    var makeTriangle = new isTriangle;
+    assert.equal(makeTriangle.theTriangle(3,3,3), "This is an equilateral triangle");
```

Comment:
again, be mindful when choosing variables so that your method calls are easy to read and make logical sense 👍